### PR TITLE
Fix VTR Parser - NoC Flow File

### PR DESCRIPTION
**Pull request by amin1377**
This PR addresses compatibility issues between the vtr_parser script and golden results files for NoC benchmarks, and the output generated by the run_vtr_task function. The updates ensure that the vtr_parser script works correctly with the latest version of run_vtr_task, and that the golden results files accurately reflect the expected output.

@vaughnbetz: Hopefully this will be the last fix on the NoC parser!

### CODE REVIEW

**Comment by vaughnbetz** [comment]
Looks like there's one more thing to fix (python lint, too long a subroutine):

************* Module vtr.task
vtr_flow/scripts/python_libs/vtr/task.py:329:0: R0915: Too many statements (60/50) (too-many-statements)

**Comment by vaughnbetz** [comment]
Merging this, but please fix the python lint failure:

************* Module vtr.task
vtr_flow/scripts/python_libs/vtr/task.py:329:0: R0915: Too many statements (60/50) (too-many-statements)

File: vtr_flow/scripts/python_libs/vtr/task.py [hunk 1]
```diff
@@ -325,6 +325,123 @@ def find_longest_task_description(configs):
     return longest
 
 
+def get_work_dir_addr(arch, circuit, noc_traffic):
+    work_dir = None
```
Comment (vaughnbetz) [inline vtr_flow/scripts/python_libs/vtr/task.py:329]: add a short comment. What does this do, and why do we need it?

**Comment by amin1377** [comment]
@vaughnbetz: CI tests are clean now :)

**Pull request merged**